editor/screen.go

```diff
@@ -58,11 +58,21 @@ type Screen struct {
 	keyShift        core.Qt__Key
 }
 
+var tooltip *widgets.QLabel
```

Comment:
I think it's better to make tooltip struct Screen's filed.

Reply:
Thank you for pointing out. 
Actually I started implementing it Screen's field at first, but I got the following error when `qtdeploy` and could not solve it. 
Would you point out if there is an idea for this error?
``` qtdeploy
../../editor/moc.go:724: ptr.QLabel_PTR undefined (type *Screen has no field or method QLabel_PTR)
../../editor/moc.go:731: ptr.QLabel_PTR undefined (type *Screen has no field or method QLabel_PTR)
../../editor/moc.go:753: unknown field 'QLabel' in struct literal of type Screen
../../editor/moc.go:1000: tmpValue.ConnectDestroyed undefined (type *Screen has no field or method ConnectDestroyed)
../../editor/moc.go:1013: tmpValue.ConnectDestroyed undefined (type *Screen has no field or method ConnectDestroyed)
```

Reply:
try to run ```qtmoc``` first

Reply:
I tried it already, but the same situation.

I also tried deleting the .moc file before qtmoc, but I got the same error  :(

Reply:
I tried it myself and it was fine. Don't bother then. I can change this after merging.

Reply:
I appreciate it. I will check the code again when it is merged.